candidate_host.go

```diff
@@ -19,6 +19,7 @@ type CandidateHostConfig struct {
 	Address     string
 	Port        int
 	Component   uint16
+	TCPType     TCPType
```

Comment:
I worry that a reader would assume ALL `CandidateHost` instances must have a `TCPType`

Do you think it would be obnoxious to have a dedicated TCP Candidate type? Or just some other way to lay it out so users understand `udp OR tcp ... if tcp then one of these types`

Reply:
I changed the type to `int` with a default value `TCPTypeUnspecified`.